pioneers-testnet/pioneers-exercise-2.md

```diff
@@ -0,0 +1,170 @@
+
+### Exercises
+
+1. Make sure that you do not have an old instance of the node running:
+
+```bash
+$ killall cardano-node
+```
+
+Start a new instance of the node, as you did in Exercise 1:
+
+```bash
+$ cardano-node run --config …
+```
+
+Verify that your new node instance is running:
+
+```bash
+$ ps x | grep cardano-node
+10765 pts/4 R + 1:20  cardano-node …
+$
+```
+
+Your node should be connected to the Pioneer Testnet and verifying the blocks that it receives.
+
+2. 	Verify that you have received some test Ada at the address that you provided to IOHK in Exercise 1, *myaddr*.
+
+```
+$ cardano-cli shelley query filtered-utxo \
+ 	--address ... \
+ 	--network-magic …
+```
+
+Create a new address *myaddr2*:
+
+```bash
+$ cardano-cli shelley address key-gen …
+```
+
+3. Build a transaction to transfer Ada from *myaddr* to *myaddr2*.  You will need to use the Shelley transaction processing operations:
+
+```bash
+$ cardano-cli shelley transaction build-raw \
+  --tx-body-file txbody …
+```
+
+This is the most basic form of transaction construction.  We will use more sophisticated ones later.  The transaction will be created in the file txbody. You will need to provide explicit transaction inputs and outputs.
+
+| Id#Index     | This identifies the UTxO that is the source of the Ada – you should get this from  *myaddr*. |
+|--------------|--------------------------------------------------------------------------------------------|
+| Out+lovelace | Hex encoded address that will receive the Ada and the amount to send in Lovelace.          |
+
+You will also need to give it a time to live, in slots (ttl) and a fee (in lovelace).  Use the following settings:
+
+| ttl 	| 1000    	|
+|-----	|---------	|
+| fee 	| 1000000 	|
+
+The settings that are used here indicate that the transaction should be processed within 1,000 slots and  that it will cost no more than 1 Ada to submit (a safe value for a simple transaction on the Testnet).  You must pay this (small) fee every time you successfully process a transaction on the Cardano Blockchain.  It will be distributed as part of the pool rewards.  The source of this fee is encoded in the transaction as a UTxO.  We are now ready to sign the transaction and submit it to the chain.
+
+4. Generate a signing key, txsign.
+
+```bash
+$ cardano-cli keygen …
+```
+
+Sign your transaction in txbody using the signing key for *myaddr*:
+
+```bash
+$ cardano-cli shelley transaction sign \
+ 	--tx-body-file txbody \
+ 	--signing-key-file txsign \
+ 	--network-magic … \
+ 	--tx-body-file txbody
+```
+
+You will need to give the correct Network Magic Id for the Testnet, as supplied by IOHK in the Genesis file (e.g. 42).
+
+5. Confirm that your node is synced, and then submit your transaction to the Blockchain:
```

Comment:
How can we `Confirm that your node is synced` !? There are no instructions for this very precise and important step.